wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+Một lĩnh vực khác đã tốn rất nhiều công sức là bảo tồn lịch sử — đảm bảo rằng các bản đồ nhịp hoạt động chính xác như bình thường, bao gồm cả các trường hợp biên không được lên kế hoạch ban đầu. osu! là một hệ sinh thái sôi động và người dùng đã tự do mở rộng trò chơi vượt xa phạm vi dự kiến của nó và chúng tôi đang cố gắng hết sức để nắm bắt và hỗ trợ điều này trong tương lai.
+
+Cuối cùng, không giống như lần lặp lại trước, chúng tôi đang dành thời gian và sự siêng năng để đảm bảo cơ sở mã sẽ phục vụ tốt cho chúng tôi trong tương lai. Chúng tôi đã hoàn thành nền tảng để cho phép các tính năng mới xuất hiện trực tuyến với tốc độ chóng mặt trong tương lai. Điều này sẽ bao gồm các thành phần giao diện người dùng mới, các cách mới để tạo giao diện cho trò chơi, hệ thống nhiều người chơi mới và đừng quên khả năng tải và phát tất cả các bản đồ nhịp hiện có của bạn trên các chế độ trò chơi hoàn toàn mới (còn gọi là bộ quy tắc)!
+
+#### Còn gì nữa?
+
+Chúng tôi có một lượng lớn các tính năng và cải tiến do người dùng yêu cầu mà chúng tôi sẽ tiếp tục phát triển với tốc độ ánh sáng. Đối với những người đã tham gia với chúng tôi gần đây và chưa trải nghiệm động lực của osu! phát triển, chuẩn bị để được ở trong một bất ngờ.
+
+#### Làm cách nào để truy cập thư mục bài hát?
+
+Không có thư mục bài hát trong lazer! Điều này cho phép chúng tôi làm những điều thú vị như không yêu cầu nhấn `F5` khi chọn bài hát để làm mới beatmap (vì beatmap luôn ở trạng thái tốt) và giảm 20–40% dung lượng ổ đĩa được sử dụng bởi beatmap. Bạn có thể đọc thêm về [cách lazer lưu trữ tệp](/wiki/Client/Release_stream/Lazer/File_storage).
+
+Nếu bạn cần thay đổi beatmap, vui lòng sử dụng trình chỉnh sửa. Trong tương lai, chúng tôi sẽ giới thiệu một chế độ trong trình chỉnh sửa giúp thư mục của beatmap có thể truy cập tạm thời để chỉnh sửa bên ngoài. Điều này sẽ cho phép bạn sử dụng các công cụ bên ngoài trên beatmap trong quá trình tạo.
+
+#### Bây giờ "osu!direct" đã có sẵn cho tất cả người chơi, liệu những người ủng hộ có bất kỳ lợi ích mới nào không?
+
+Một số bộ lọc trong danh sách beatmap vẫn chỉ dành cho người hỗ trợ.
+
+Ngoài ra còn có một số lợi ích bổ sung:
+
+- Những người ủng hộ có thể tạo danh sách phát kéo dài hơn
+
+Chúng tôi dự định sẽ xem xét các lợi ích mới trong tương lai, nhưng trọng tâm của chúng tôi hiện tại là tính năng tương đương với tính năng ổn định, vì vậy vui lòng sử dụng giao dịch mua thẻ người ủng hộ của bạn như một cách để... hỗ trợ sự phát triển của trò chơi!
+
+#### Nếu tôi gian lận trên lazer thì tôi có bị  không?
+
+Đương nhiên rồi.
+
+#### Nếu tôi phát hiện ai đó gian lận trên lazer tôi nên báo cáo họ như thế nào?
+
+Y như cách mà bạn vẫn hay làm thôi.
+
+#### Giao dịch vi mô ở đâu?
```

Comment:
Wait for other suggestion, but definitely not `vi mô`.

Reply:
I know

Reply:
```suggestion
#### Liệu tôi có bị lùa gà về độ miễn phí của trò chơi không?
```
idk what i just wrote thanks tanamonomae13 lmao im not gonna read all of these
maybe i will read it one day please tell me if you need anything